src/cctag/CCTag.hpp

```diff
@@ -392,8 +389,8 @@ class CCTag : public ICCTag
   std::vector< std::vector< DirectedPoint2d<Eigen::Vector3f> > > _points;
   Eigen::Matrix3f _mHomography;
   float _quality;
-  int    _pyramidLevel;
-  float _scale;
+  int    _pyramidLevel{};
+  float _scale{};
   int    _status;
```

Comment:
If you start adding initializers on some member variables, it would be good to do it for all members of the class.